scale/ethdebug.go

```diff
@@ -0,0 +1,24 @@
+package scale
```

Comment:
Thinking about putting it into `metrics/ethdebug`. What do you think?

Reply:
let's move when it will be used anywhere outside of scale package